Microsoft.DotNet.Interactive.PowerShell/Modules/Microsoft.DotNet.Interactive.PowerShell/Microsoft.DotNet.Interactive.PowerShell.psd1

```diff
@@ -0,0 +1,131 @@
+#
+# Module manifest for module 'Microsoft.DotNet.Interactive.PowerShell'
+#
+# Generated by: tyleonha
+#
+# Generated on: 1/23/2020
+#
```

Comment:
Replace this comment section with the following?
```
//
// Copyright (c) Microsoft. All rights reserved.
// Licensed under the MIT license. See LICENSE file in the project root for full license information.
//
```

Reply:
The copyright and license we're using elsewhere is:

```
// Copyright (c) .NET Foundation and contributors. All rights reserved.
// Licensed under the MIT license. See LICENSE file in the project root for full license information.
```

Reply:
updated 👍 Not sure what you want the AuthorName and CompanyName to be in the psd1 so I left it at Microsoft for now. Let me know if you prefer something else.

Reply:
Should probably be `.NET Foundation` instead of `Microsoft`